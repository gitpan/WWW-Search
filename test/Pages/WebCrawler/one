http://www.webcrawler.com/cgi-bin/WebQuery?perPage=25&start=0&search=disestablishmentarianism&showSummary=true
