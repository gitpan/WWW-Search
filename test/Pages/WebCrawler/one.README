This set of test-result pages was created on Fri Oct  8 11:07:56 1999
