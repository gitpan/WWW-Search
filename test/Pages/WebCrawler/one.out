http://members.tripod.com/~tritemp/index.htm
http://www.angelfire.com/de/exposition/cyberart.html
